tests/components/plugwise/test_init.py

```
@@ -229,6 +230,27 @@ async def test_migrate_unique_id_relay(
         hass, mock_config_entry, entitydata, old_unique_id, new_unique_id
     )
 
+async def test_entry_migration(hass: HomeAssistant, snapshot: SnapshotAssertion) -> None:
+    """Test config entry version 1 -> 2 migration."""
+    mock_config_entry = MockConfigEntry(
+        title="My Plugwise",
+        domain=DOMAIN,
+        data={
+            CONF_HOST: "127.0.0.1",
```

Ruff

Undefined name `CONF_HOST`

(F821)